python/requirements.txt

```diff
@@ -1,3 +1,7 @@
 numpy>=1.19.5, <=1.26
+pybind11>=2.9.0, <=2.10.3
 PyYAML>=5.3.1, <=6.0.1
-dataclasses>=0.6, <=0.8
\ No newline at end of file
+dataclasses>=0.6, <=0.8
```

Comment:
?

Reply:
no clue why this got lit up - maybe space at EOL?